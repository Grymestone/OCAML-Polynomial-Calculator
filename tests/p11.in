(2a + 3a^6 + a^1)^2
