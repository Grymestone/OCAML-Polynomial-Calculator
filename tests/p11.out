((((2*a)+(3*(a)^6))+(a)^1))^2
18x^7 + 9x^12 + 9x^2